packages/react-native/README.md

```diff
@@ -78,3 +137,20 @@ DashX.track('event_name', { hello: 'world' } /* Event data */);
 ```javascript
 DashX.reset();
 ```
+
+### Set Identity Token
+
+In order to subcribe to push notifications you need to set identity token like so
+
+```javascript
+DashX.setIdentityToken('your_public_key');
```

Comment:
The placeholder text ('your_public_key') is misleading. This section needs more explanation.

Reply:
Ah, this should be `user's_identity_token`